src/SFA.DAS.ProviderApprenticeshipsService.Domain/Http/IHttpClientWrapper.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using SFA.DAS.NLog.Logger;
+
+namespace SFA.DAS.ProviderApprenticeshipsService.Domain.Http
+{
+    public interface IHttpClientWrapper
+    {
+        Task<string> SendMessage<T>(T content, string url);
+        Task<T> Get<T>(string authToken, string url);
+        Task<string> GetString(string url, string accessToken);
+        string BaseUrl { get; set; }
+        List<MediaTypeWithQualityHeaderValue> MediaTypeWithQualityHeaderValueList { get; set; }
+        string AuthScheme { get; set; }
+    }
+
+    public class HttpClientWrapper : IHttpClientWrapper
```

Comment:
All of the HTTP stuff should be available from a shared library.

Reply:
sure I'll take a look at sorting this, it did feel a bit odd having to paste all this stuff in